scripts/Tools/case.submit

```diff
@@ -115,7 +116,12 @@ sub main
 	    check_DA_settings();
 	    `./xmlchange --noecho  RUN_WITH_SUBMIT=TRUE`;
 	    $config{RUN_WITH_SUBMIT}='TRUE';
-	    $batchutils->submitJobs();
+            if (defined $opts{'jobid'}) {
```

Comment:
I think that this was added because acme removed the requirement to use the case.submit script to start the job and so they are doing manually what that script is designed to automate.